LinqToDBPostGisNetTopologySuite/SpatialRelationships.cs

```diff
@@ -194,5 +194,53 @@ public static class SpatialRelationships
         {
             throw new InvalidOperationException();
         }
+
+        /// <summary>
+        /// For geometry type returns true if the 3d distance between two objects is within distance_of_srid specified projected units (spatial ref units)
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_3DDWithin.html
+        /// </remarks>
+        /// <param name="geom1">Input geometry 1</param>
```

Comment:
Parameter names `geometry` and `other` would be better ?

Reply:
> Parameters names `geometry` and `other` instead of `geom1`, `geom2` would be better ?

GISer often use `geom` instead of `geometry` as a simplified word ，especially those who use PostGIS.
I thikin it would be better,

Reply:
Well, I'll review it later along with other methods, just want to keep naming style uniform across all classes (the PostGIS Reference naming isn't uniform actually)

Reply:
> Parameters names `geometry` and `other` instead of `geom1`, `geom2` would be better ?

It's better i think.

Reply:
> Well, I'll review it later along with other methods, just want to keep naming style uniform across all classes (the PostGIS Reference naming isn't uniform actually)


* `geometry` and `other` have better readability.
* `Geometry` only has 8 characters , it's not wordy.

I'd love to vote for `geometry`.
Anyway,which style you choose i will take  :)

Reply:
The `distance` parameter need to fix too:

````c#
<param name="distance">Distance (in SRID units).</param>
````

Looks much better.